akka-http-core/src/test/scala/akka/http/scaladsl/TightRequestTimeoutSpec.scala

```diff
@@ -36,19 +36,19 @@ class TightRequestTimeoutSpec extends AnyWordSpec with Matchers with BeforeAndAf
   "Tight request timeout" should {
 
     "not cause double push error caused by the late response attempting to push" in {
-      val (hostname, port) = SocketUtil.temporaryServerHostnameAndPort()
       val slowHandler = Flow[HttpRequest].map(_ => HttpResponse()).delay(500.millis.dilated, OverflowStrategy.backpressure)
-      val binding = Http().bindAndHandle(slowHandler, hostname, port)
+      val binding = Http().bindAndHandle(slowHandler, "localhost", 0).awaitResult(2.seconds.dilated)
```

Comment:
would be nice to have an `awaitResult` with a default pause, looks a bit crowded now

Reply:
Good point, went back to using scalatest's `futureValue` here.